### DISCUSSION TITLE
CANVA

### DISCUSSION THREAD

**Discussion by bariou35** [community/community#39821] [New to GitHub]
### Select Topic Area

Question

### Body

Est-il possible d'avoir l'application ou le site "CANVA" gratuitement ?

**Comment by ghostinhershell** [comment]
Hi @bariou35 👋 

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try reaching out to Canva's support. The GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

With that being said, we take our Code of Conduct very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.

Best of luck!
